src/control/control.h

```diff
@@ -229,14 +230,8 @@ class ControlDoublePrivateConst : public ControlDoublePrivate {
   public:
     ~ControlDoublePrivateConst() override = default;
 
-    void setInner(double value, QObject* pSender) override {
-        Q_UNUSED(value)
-        Q_UNUSED(pSender)
+  private:
+    void setInner(double, QObject*) override {
         DEBUG_ASSERT(!"Trying to modify a default constructed (const) control object");
     };
```

Comment:
adhering to [cpp core guidelines "F.9: Unused parameters should be unnamed"](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rf-unused). I personally prefer that because it does not have the risk where a parameter is marked with `Q_UNUSED` but still used in the body somewhere later.